src/Button/styles.ts

```diff
@@ -291,6 +295,11 @@ export const getButtonStyles = (theme?: Theme) => {
     '&[data-block="block"]': {
       width: '100%',
     },
+    '&[data-inactive]:not([disabled]):not([aria-disabled])': {
+      backgroundColor: `var(--button-inactive-bgColor, ${theme?.colors.btn.inactive.bg})`,
+      border: 0,
```

Comment:
So having a `border:0` affects the width of the button. The disabled button however has the border.